app/controllers/bookings_controller.rb

```diff
@@ -1,2 +1,21 @@
 class BookingsController < ApplicationController
+  def new
+    @booking = Booking.new
+  end
+
+  def create
+    @booking = Booking.new(booking_params)
+    @booking.user = current_user
+
+    if booking.save
+      redirect_to @booking, notice: "Booking was successfully created."
```

Comment:
maybe should be redirect to bookings_path ?

Reply:
Now the booking is redirected to theme show.